static/postgres/v3-chinook-ndc-metadata/native_queries/summarize_organizations.sql

```diff
@@ -0,0 +1,22 @@
+SELECT
```

Comment:
Ah, how I wished I could have had a separate file when I made this query. Boy is it easier on the eyes without string escaping~